dGame/dUtilities/SlashCommandHandler.cpp

```diff
@@ -1653,7 +1655,7 @@ void SlashCommandHandler::HandleChatCommand(const std::u16string& command, Entit
 		}
 	}
 
-	if (chatCommand == "spawnphysicsverts" && entity->GetGMLevel() >= 6) {
+	if (chatCommand == "spawnphysicsverts" && entity->GetGMLevel() >= eGameMasterLevel::LEAD_MODERATOR) {
```

Comment:
This is off by one - the current version is compared with 6 which is junior developer